File: text/ringo/create/text_create.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	text_create.js
//
//						Dec/27/2013
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");
var args = require('system').args;
var file_out = args[1];

var dict_aa = data_prepare_proc ();
text_manipulate.dict_display_proc (dict_aa);
text_manipulate.text_write_proc (file_out,dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------
function data_prepare_proc ()
{
	var dict_aa = new Object ();

	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2381','名古屋',91865,'1953-8-21');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2382','豊橋',23157,'1953-10-15');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2383','岡崎',68741,'1953-5-2');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2384','一宮',38714,'1953-6-22');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2385','蒲郡',42358,'1953-8-14');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2386','常滑',69142,'1953-9-12');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2387','大府',36281,'1953-3-21');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2388','瀬戸',58246,'1953-7-26');
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t2389','犬山',27451,'1953-10-2');

	return	dict_aa;
}

// ------------------------------------------------------------------

File: text/octave/create/text_create.m
#! /usr/bin/octave -qf
#
#	text_create.m
#
#					Nov/11/2014
#
# ------------------------------------------------------------------
path (path, "/var/www/data_base/common/octave_common");
# ------------------------------------------------------------------
function unit_aa = unit_define_proc (name,population,date_mod)
	unit_aa.name = name;
	unit_aa.population = population;
	unit_aa.date_mod = date_mod;
endfunction
# ------------------------------------------------------------------
function dict_aa = data_prepare_proc ()
	dict_aa = struct;
	dict_aa.("t2381") = unit_define_proc ("名古屋","72961","1948-4-30");
	dict_aa.("t2382") = unit_define_proc ("豊橋","48915","1948-5-10");
	dict_aa.("t2383") = unit_define_proc ("岡崎","27384","1948-6-14");
	dict_aa.("t2384") = unit_define_proc ("一宮","36912","1948-9-9");
	dict_aa.("t2385") = unit_define_proc ("蒲郡","42138","1948-8-4");
	dict_aa.("t2386") = unit_define_proc ("常滑","35187","1948-1-21");
	dict_aa.("t2387") = unit_define_proc ("大府","76125","2014-12-31");
	dict_aa.("t2388") = unit_define_proc ("瀬戸","25791","1948-10-26");
	dict_aa.("t2389") = unit_define_proc ("犬山","73482","1948-2-15");
endfunction

# ------------------------------------------------------------------
printf ("*** 開始 ***\n");
arg_list = argv ();
file_out = arg_list{1};
#
dict_aa = data_prepare_proc ()
#
text_write_proc (file_out,dict_aa);
#
printf ("*** 終了 ***\n");
# ------------------------------------------------------------------
